metric-anomaly-data-model/src/main/proto/org/hypertrace/alert/engine/eventcondition/config/service/v1/metric_anomaly_event_condition.proto

```diff
@@ -0,0 +1,104 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.alert.engine.eventcondition.config.service.v1;
+
+message MetricAnomalyEventCondition {
+  MetricSelection metric_selection = 1;
+  repeated ViolationCondition violation_condition = 2;
+}
+
+message MetricSelection {
+  string key = 1;
+  string scope = 2;
+  MetricAggregationFunction metric_aggregation_function = 3;
+  string metric_aggregation_interval = 4; //iso duration string PT15S
+  repeated Filter filters = 5;
+}
+
+enum MetricAggregationFunction {
+  METRIC_AGGREGATION_FUNCTION_TYPE_UNSPECIFIED = 0;
+  METRIC_AGGREGATION_FUNCTION_TYPE_SUM = 1;
+  METRIC_AGGREGATION_FUNCTION_TYPE_AVG = 2;
+}
+
+message Filter {
+  oneof filter {
+    LeafFilter leaf_filter = 1;
+    CompositeFilter composite_filter = 2;
+  }
+}
+
+message LeafFilter {
+  LhsExpression lhs_expression = 1;
+  Operator operator = 2;
+  RhsExpression rhs_expression = 3;
+}
+
+enum Operator {
```

Comment:
Another benefit of separating the leaf and composite filters - they work in different operators. We can separate the value operators and logical operators into two different enums.

Reply:
Done.